src/js/common/components/RequiredLoginView.jsx

```diff
@@ -113,7 +134,23 @@ class RequiredLoginView extends React.Component {
       }
     } else if (this.props.authRequired === 3) {
       if (this.state.accountType === 3) {
-        view = this.props.children;
+        if (this.state.isServiceOffline) {
+          // If there is nothing in va_profile, show system down message.
+          view = <SystemDownView messageLine1="We're sorry. The application you are looking for is currently down while we fix a few things. We will be back up as soon as we can."/>;
```

Comment:
Could we tweak this slightly to:
"Sorry, this application is temporarily down while we fix a few things. We'll get it back up as soon as possible."